src/kernel/chainparams.cpp

```diff
@@ -395,10 +395,10 @@ class CTestNet4Params : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from RPC: getchaintxstats 4096 000000005be348057db991fa5d89fe7c4695b667cfb311391a8db374b6f681fd
-            .nTime    = 1723651702,
-            .tx_count = 757229,
-            .dTxRate  = 0.01570402633472492,
+            // Data from RPC: getchaintxstats 4096 00000000798bd720800263b69bee01aae57ea376596415a04423da29f000c367
```

Comment:
In 70e294c0930a412fb6945eb753d1a39ef52fb324 "[kernel] update chainTxData for v29"

```
$ bitcoin-cli -testnet4 getchaintxstats 4096 00000000798bd720800263b69bee01aae57ea376596415a04423da29f000c367
error code: -8
error message:
Block is not in main chain
```

Reply:
fixed